packages/webapp/src/components/Square/index.jsx

```diff
@@ -52,7 +52,9 @@ export default function Square({ color = 'active', children, isCropTile, ...prop
       className={clsx(classes.container, classes[color], isCropTile && classes.cropTile)}
       {...props}
     >
-      {color === 'needsPlan' ? <FaExclamation style={{ fontSize: '12px' }} /> : children}
+      {/* commenting for the future use */}
+      {/* {color === 'needsPlan' ? <FaExclamation style={{ fontSize: '12px' }} /> : children} */}
```

Comment:
I'm not sure we need to keep this comment since one of the points of this ticket seems to be to remove the exclamation mark.